examples/Storage/Storage.Tests.ps1

```diff
@@ -88,6 +88,7 @@ Describe 'Verify Storage Container' {
       ResourceGroupName = 'rg-test'
       AccountName       = 'azbenchpressstorage'
       Name              = 'noazbenchpresscontainer'
+      ErrorAction       = 'SilentlyContinue'
```

Comment:
With this, be sure to add the `-ErrorAction SilentlyContinue` comment.

Reply:
The comment is already there, right before calling the cmdlet :)

Reply:
Oh man, my bad. I looked above it and GH didn't show the lines below while reviewing.

Reply:
Haha no worries! Totally understandable